tmt/convert.py

```diff
@@ -272,6 +272,13 @@ def read(path, makefile, nitrate, purpose, disabled):
             echo(style('duration: ', fg='green') + data['duration'])
         except AttributeError:
             pass
+        # Environment
+        env_vars = re.findall(r'^Environment:\s*(.*)', testinfo, re.M)
+        if env_vars:
+            data['environment'] = tmt.utils.shell_to_dict([
+                 env_var for line in env_vars for env_var in line.split()])
```

Comment:
Why do you split the line on whitespace? What if the variable contains spaces?

Reply:
Confirmed this is broken:

    @echo "Environment:     TEST=one two three" >> $(METADATA)

results in:

    Invalid variable specification 'two'.

Reply:
hm, if I pass the variable as it is (`['AVC_ERROR=one two]`) it gives me same error. How should I pass it so `_add_simple_var()` makes one string with space out of it @psss ?

Reply:
Yeah, that's why I suggested to do a simple split on the `=` character (or regexp match for `Environment: KEY=VAL` line) instead of using the `shell_to_dict` function. It would work with `TEST='one two three'` but that is not expected in `Makefile` / `testinfo.desc`.

Reply:
Fixed in 81ca9b2